### PR TITLE
Automation of release workflow

### LINKED ISSUE #1137: automation of release for quick version upgrade and bug hotfix
ideally, it should be an automation process triggered by the tag push.

### PR DESCRIPTION
**Pull request by lidongze0629**
## What do these changes do?

- [x] Release images for GraphScope and GraphScope-Store
- [x] Release python wheel packages (macos and linux)
- [x] Release helm charts for  GraphScope and GraphScope-Store 

## Related issue number

Fixes #1137

### CODE REVIEW

File: .github/workflows/build-graphscope-wheels-linux.yml [hunk 1]
```diff
@@ -78,6 +105,21 @@ jobs:
...
+      if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') }}
+      run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/v}
+
+    - name: Release Images
+      if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') && github.repository == 'alibaba/GraphScope' }}
+      run: |
+        echo ${{ secrets.ALIYUN_TOKEN }} | sudo docker login --username=${{ secrets.ALIYUN_USER }} registry.cn-hongkong.aliyuncs.com --password-stdin
+        sudo docker tag graphscope/graphscope:${SHORT_SHA} \
+            registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:${{ steps.tag.outputs.TAG }}
+        sudo docker push registry.cn-hongkong.aliyuncs.com/graphscope/graphscope:${{ steps.tag.outputs.TAG }}
```
Comment (siyuan0322) [inline .github/workflows/build-graphscope-wheels-linux.yml:122]: Tag it to `registry-vpc.xxx` then push can reduce the amount of time.

File: .github/workflows/build-graphscope-wheels-linux.yml [hunk 2]
```diff
@@ -51,8 +81,28 @@ jobs:
...
 
+    # We do this, since failures on test.pypi aren't that bad
+    - name: Publish to Test PyPI
+      # if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v') && github.repository == 'alibaba/GraphScope' }}
+      # if: ${{ github.ref == 'refs/heads/main' && github.repository == 'alibaba/GraphScope' }}
+      uses: pypa/gh-action-pypi-publish@v1.4.2
+      with:
+        user: __token__
+        password: ${{ secrets.TEST_PYPI_PASSWORD }}
+        repository_url: https://test.pypi.org/legacy/
```
Comment (sighingnow) [inline .github/workflows/build-graphscope-wheels-linux.yml:92]: But it is still uninstallable?

File: .github/workflows/build-graphscope-wheels-linux.yml [hunk 3]
```diff
@@ -79,6 +129,27 @@ jobs:
         cd ${GITHUB_WORKSPACE}
         sudo make graphscope-image
 
+    - name: Extract Tag Name
+      id: tag
+      env:
+         GITHUB_REF_DUMMY: refs/tags/v0.9.1
```
Comment (sighingnow) [inline .github/workflows/build-graphscope-wheels-linux.yml:135]: v0.9.1 looks strange here. Could it be "nightly" or something like that?

File: VERSION [hunk 4]
```diff
@@ -1 +1 @@
-0.9.0
+0.9.1
```
Comment (sighingnow) [inline VERSION:1]: I strongly suggest not to mix unrelated logic to a single PR.

File: .github/workflows/build-graphscope-wheels-linux.yml (hunk 3 above)
Comment (lidongze0629) [reply .github/workflows/build-graphscope-wheels-linux.yml:135]: Just for test release workflow

File: VERSION (hunk 4 above)
Comment (lidongze0629) [reply VERSION]: Done

**Review by sighingnow — APPROVED** [review]

**Review by yecol — APPROVED** [review]

**Comment by siyuan0322** [comment]
:trollface:

**Pull request merged**